rust/otap-dataflow/crates/otap/src/fake_data_generator/fake_signal.rs

```diff
@@ -136,6 +137,11 @@ fn spans(signal_count: usize, registry: &ResolvedRegistry) -> Vec<Span> {
                 )
                 .kind(otel_span_kind(group.span_kind.as_ref()))
                 .end_time_unix_nano(end_time)
+                // Note: without status set, the OTAP encoder fails at runtime
+                .status(otap_df_pdata::proto::opentelemetry::trace::v1::Status::new(
+                    StatusCode::Ok,
+                    "ok",
+                ))
```

Comment:
possibly fixed by @jmacd in https://github.com/open-telemetry/otel-arrow/pull/1436 ?